cmd/storj-sim/network.go

```diff
@@ -250,7 +250,7 @@ func newNetwork(flags *Flags) (*Processes, error) {
 				"--server.address", process.Address,
 				"--server.private-address", net.JoinHostPort(host, port(satellitePeer, i, privateGRPC)),
 
-				"--server.extensions.revocation=false",
+				"--server.extensions.revocation=false", //todo update to true
```

Comment:
should this say "update to true when we want to use revocation" or something?

Reply:
yeah I could add that. I'm wondering if we want to keep revocationdb once we have a redis namespace package so we can make sure it works with the satellite separation?

Reply:
Just fyi, this config flag only determines [whether we check for revocation extensions on peer certificates during TLS handshakes](https://github.com/storj/storj/blob/master/pkg/peertls/tlsopts/options.go#L107), which is different from whether we're opening or [passing a revocation DB to the server run method](https://github.com/storj/storj/blob/94cec7e3b05b06b522697e206d6e29b6b45156ee/pkg/server/config.go#L26).

Also, just curious what you mean by "wondering if we want to keep revocationdb..."?

Reply:
oh ok good to know. and yeah that makes little sense now that I re-read that haha... I meant if we wanted to keep that flag=true.